README.md

```diff
@@ -1,4 +1,12 @@
+<h1 align="center">
+    <picture>
+        <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/7659/174594540-5e29e523-396a-465b-9a6e-6cab5b15a568.svg">
+        <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/7659/174594559-0b3ddaa7-e75b-4f10-9dee-b51431a9fd4c.svg">
+        <img src="https://user-images.githubusercontent.com/7659/174594540-5e29e523-396a-465b-9a6e-6cab5b15a568.svg" alt="Dependabot" width="336">
+    </picture>
+</h1>
+
 # Dependabot Demo Repository
 
```

Comment:
// FindOptions are for fetching Artifact(s) out of the scope of the current run.
// Must specify a PAT with actions:read scope for cross run/repo lookup otherwise these will be ignored.
export interface FindOptions {
  findBy?: {
    // Token with actions:read permissions
    token: string
    // WorkflowRun of the artifact(s) to lookup
    workflowRunId: number
    // Repository owner
    repositoryOwner: string
    // Repository name
    repositoryName: string
  }
}

Reply:
# - **@[]() **